electron/main.ts

```diff
@@ -0,0 +1,51 @@
+// Modules to control application life and create native browser window
+const {app, BrowserWindow} = require('electron')
```

Comment:
I am getting this warning:

`TS1208: 'main.ts' cannot be compiled under '--isolatedModules' because it is considered a global script file. Add an import, export, or an empty 'export {}' statement to make it a module.`

Could you please fix it?